content/en/blog/2024/v1-apis/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Getting Started with using v1 Istio APIs "
```

Comment:
Bit of a nit, but if we're considering [pushing new users toward Gateway API](https://github.com/istio/istio.io/issues/14975) and Ambient, maybe we should use a phrase other than "getting started" here? Not a strong opinion though, could also be fine to leave as-is.

Reply:
Updated to Introducing Istio v1 APIs